experimental/tfhub/rikai/experimental/tfhub/tfhub_registry.py

```diff
@@ -17,34 +17,39 @@
 
 import tensorflow_hub as tfhub
 
-from rikai.internal.reflection import find_func, has_func
+from rikai.internal.reflection import has_func
 from rikai.logging import logger
-from rikai.spark.sql.codegen.base import Registry, udf_from_spec
+from rikai.spark.sql.codegen.base import Registry
 from rikai.spark.sql.model import ModelSpec
+from rikai.spark.sql.model import is_fully_qualified_name
 
 
 class TFHubModelSpec(ModelSpec):
-    def __init__(self, handle, raw_spec: ModelSpec):
+    def __init__(self, publisher, handle, raw_spec: "ModelSpec"):
         spec = {
             "version": "1.0",
-            "schema": raw_spec["schema"],
-            "model": {"flavor": raw_spec["flavor"], "uri": raw_spec["uri"]},
-            "transforms": {
-                "pre": raw_spec.get("preprocessor", None),
-                "post": raw_spec.get("postprocessor", None),
+            "schema": raw_spec.get("schema", None),
```

Comment:
invoked via `apply_model_spec` for testing purpose, `schema`/`flavor`/`modelType` might be missing

that's why I'm using `raw_spec.get("schema", None)`. For raw_spec from the Scala case class, we do not fill the default None value.